### PR TITLE
B3 logistics

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by kkrampa** [comment]
@biyeun

File: custom/ilsgateway/tanzania/reports/facility_details.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from corehq.apps.locations.models import SQLLocation
 from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
 from corehq.apps.sms.models import SMS
+from corehq.apps.users.models import CommCareUser
```
Comment (dimagimon) [inline custom/ilsgateway/tanzania/reports/facility_details.py]: F401 'CommCareUser' imported but unused

File: custom/ewsghana/templates/ewsghana/reporting_rates_print_report.html [hunk 2]
```diff
@@ -45,9 +42,11 @@
     {% compress css %}
     <link type="text/less" rel="stylesheet" media="all" href="{% static 'style/less/hqstyle-core.less' %}"/>
```
Comment (biyeun) [inline custom/ewsghana/templates/ewsghana/reporting_rates_print_report.html]: this is the old stylesheet for b2, curious why it's still here?

**Comment by kkrampa** [comment]
@biyeun Updated

**Pull request merged**